src/platform/forms-system/src/js/components/FormNav.jsx

```diff
@@ -97,6 +105,8 @@ export default function FormNav(props) {
           {!showHeader && (
             <div className="vads-u-font-size--h4">{stepText}</div>
           )}
+          {subTitle && <h3>{subTitle}</h3>}
+          {instructions && <p>{instructions}</p>}
```

Comment:
Looks like what you want to do for your form is create a [block of text on the page](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/forms/common-patterns-for-building-forms#creating-a-block-of-text-with-no-associated-fields), right?

Reply:
Oh, yes. I will try that. Thanks for pointing me to the documentation.